sublime_linter.py

```diff
@@ -304,6 +318,11 @@ def make_view_has_changed_fn(view):
     initial_change_count = view.change_count()
 
     def view_has_changed():
+        if persist.kill_switch:
```

Comment:
Maybe `persist.get('kill_switch', True)` bc otherwise we get an AttributeError here on the next upgrade?